src/app/my-journeys-listing/my-journeys-listing.page.html

```diff
@@ -0,0 +1,14 @@
+<app-application-header [config]="headerConfig" ></app-application-header>
+
+<ion-content [fullscreen]="true">
+  @if (myJourneys.length > 0) {
```

Comment:
wherever you're using statement myJourneys.length > o just use myJourney.llength it will evaluvate or did you face any issues in that?

Reply:
the suggestion will work